EE_API_automation/cico_run_EE_tests.sh

```diff
@@ -14,7 +14,8 @@ if [ -e "../jenkins-env" ]; then
 fi
 
 echo installing dependency
-sudo yum -y install python2 python2-pytest python2-requests python2-jmespath
+sudo yum -y install python2 python-pip
```

Comment:
Do we need to enable EPEL repository before installing pip? sudo yum install epel-release

Reply:
Yes, we need to enable epel...added it